rules-tests/Transform/Rector/Assign/PropertyFetchToMethodCallRector/Fixture/fixture3.php.inc

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Rector\Tests\Transform\Rector\Assign\PropertyFetchToMethodCallRector\Fixture;
+
+/**
+ * @property string foo
+ * @property string bar
+ * @method string foo
+ * @method string bar($string)
+ */
+final class Fixture3
+{
+    public $word = '';
+
+    public function word(): string
+    {
+        return 'word';
+    }
+
+    public function myFactory(Fixture3 $myClass)
+    {
+        return [
+            'word' => $myClass->word
+        ];
+    }
+
+    public function __get(string $name)
+    {
+        return 'foo';
+    }
+
+    public function __call($method, $attributes)
+    {
+        return 'foo';
+    }
+}
```

Comment:
This class should be in `/Source`, as it's not target of Rector rule, just being used.